docs/open-api/filesets.yaml

```diff
@@ -183,22 +183,23 @@ components:
       properties:
         name:
           type: string
-          description: The name of the fileset
+          description: The name of the fileset. Can not be empty.
         type:
           type: string
-          description: The type of the fileset
+          description: The type of the fileset. The value of type should be EXTERNAL or MANAGED
```

Comment:
Is it uppercase or lowcase?

Reply:
I think it should be uppercase.
![image](https://github.com/datastrato/gravitino/assets/38486782/8fe32d21-8aee-4981-be4c-6c58285671d9)

```
 Fileset fileset =
                TreeLockUtils.doWithTreeLock(
                    ident,
                    LockType.WRITE,
                    () ->
                        dispatcher.createFileset(
                            ident,
                            request.getComment(),
                            Optional.ofNullable(request.getType()).orElse(Fileset.Type.MANAGED),
                            request.getStorageLocation(),
                            request.getProperties()));
            return Utils.ok(new FilesetResponse(DTOConverters.toDTO(fileset)));
```